config.yaml

```diff
@@ -5,5 +5,5 @@ database:
   host : 192.168.1.144
 
 server:
-  host: 0.0.0.0
+  host: 192.168.1.144
```

Comment:
What exactly is this IP address? Is this local to your setup?

Reply:
Oops yes. I have somewhat of a special situation and this is why mostly this is there (and someone else may have it). I have a docker server machine at home that is different from my devel machine (the server is Linux and my dev machine is FreeBSD and FreeBSD does not support running Docker in server mode but you can run the client). I run docker from my dev machine via ```docker -H 192.168.1.144``` which means that the containers run on the actual server machine. When you run the docker FastAPI image, it binds to 0.0.0.0 and this is OK, until you try to construct URLs inside your APP, like links from one page in our case (index.html template) to a splash page for a particular bedfile. This will also probably come in handy when running the bedstat server on a machine like Neal's where the IP is internal and needs something like traefik to expose it as a public IP/port. However, for dev purposes for most people, this setting will be 0.0.0.0 and will match what FastAPI docker image chooses automatically.